src/GraphQL.Tests/MsDiContainer.cs

```diff
@@ -64,6 +64,11 @@ public T Get<T>()
             _services.AddSingleton(provider => instanceCreator());
         }
 
+        public void Scoped<TService>() where TService : class
```

Comment:
Is it worth adding to other containers?

Reply:
Ooops, there are no other containers actually, only `SimpleContainer`. And I see that `MsDiContainer` is used only in single test - `BugWithCustomScalarsInDirective` 🤔. I woud like to remove synthetic `I/SimpleContainer` and switch to `IServiceRegister` and its implementations (that we should write, of course). A bunch of work indeed.

Reply:
I assume we will track this in #2937.  resolving comment here.